### PR TITLE
Feature/#29/warnings

### PR DESCRIPTION
**Pull request by alfetopito**
# Description

Last part of #29 

Adding warnings

All Rinkeby/Mainnet warnings at once:
screenshot_2020-11-20_08-53-16

xDai warnings (no minimum bracket funding)
screenshot_2020-11-20_09-48-03

# To Test
1. Open the app

For every warning check that:

- [ ] It should be possible to deploy the strategy, given that there are no errors.

## Single bracket warning

1. On Total brackets input, insert 1

- [ ] The warning should be displayed
screenshot_2020-11-20_09-55-32

## Start price warning

1. Select two tokens from the dropdown
2. On Start price input, insert a value that is over 2% above the market price

- [ ] The warning should be displayed
screenshot_2020-11-20_09-57-40

3. Now input a values that is over 2% below the market price

- [ ] The warning should be displayed
screenshot_2020-11-20_09-58-31

## Lowest price warning

1. Insert values on Lowest price, Start price and Highest price inputs.
2. The Lowest price must be equal to or greater than Start price

- [ ] The warning should be displayed
screenshot_2020-11-20_10-01-04

## Highest price warning

1. Insert values on Lowest price, Start price and Highest price inputs.
2. The Highest price must be equal to or smaller than Start price

- [ ] The warning should be displayed
screenshot_2020-11-20_10-01-53

## Token funding warning

1. While on Mainnet or Rinkeby
2. Select two tokens from the dropdown
3. Insert values on Lowest price, Start price and Highest price inputs
4. Insert a value on funding input of any token that's equivalent to less than 5000 USD in given token

- [ ] The warning should be displayed
screenshot_2020-11-20_10-07-01

5. Insert a value on the same funding input equal to or higher than 5000 USD in given token

- [ ] The warning should no longer be displayed

6. While on xDai network, repeat steps 2 to 5

- [ ] No warning should be displayed

# Background

N/A

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 * **🔐 [Safe app link](https://pr149--safecmmappreact.review.gnosisdev.com)**: To be added as a custom app on [Safe Multisig Rinkeby](https://rinkeby.gnosis-safe.io/), [Mainnet](https://gnosis-safe.io/) or [xDai](https://xdai.gnosis-safe.io)
 * **📖 [Storybook](https://pr149--safecmmappreact.review.gnosisdev.com/storybook/)**: Component stories

**Review by andre-meyer — APPROVED** [review]
Looks good so far

File: src/components/pages/deploy/DeployPage.stories.tsx [hunk 1]
```diff
@@ -70,15 +54,15 @@ export const Default = Template.bind({});
...
 export const WithError = Template.bind({});
 WithError.parameters = {
-  recoilStates: [
-    ...defaultStates,
-    [errorAtom, { label: "Insufficient DAI balance" }],
-  ],
+  // recoilStates: [
+  //   ...defaultStates,
+  //   [warningsAtom, { label: "Insufficient DAI balance" }],
+  // ],
```
Comment (andre-meyer) [inline src/components/pages/deploy/DeployPage.stories.tsx:65]: Remove?

File: src/components/pages/deploy/DeployPage.stories.tsx [hunk 2]
```diff
@@ -87,12 +71,12 @@ const TemplateContainer = (): JSX.Element => <DeployPage />;
...
-    return async (): Promise<void> => {
-      return;
-    };
-  },
+  // useDeployStrategy: (): // params: UseDeployStrategyParams
+  // UseDeployStrategyReturn => {
+  //   return async (): Promise<void> => {
+  //     return;
+  //   };
+  // },
```
Comment (andre-meyer) [inline src/components/pages/deploy/DeployPage.stories.tsx:79]: Remove?

**Review by andre-meyer — APPROVED** [review]
Fantastic work, this looks great

**Comment by andre-meyer** [comment]
Feel free to merge

File: src/components/pages/deploy/DeployPage.stories.tsx (hunk 1 above)
Comment (alfetopito) [reply src/components/pages/deploy/DeployPage.stories.tsx:65]: Ah yes, forgot to clean up this part, thanks!

Comment (alfetopito) [reply src/components/pages/deploy/DeployPage.stories.tsx:65]: So, spent a few hours trying to mock the recoil hook.
Still haven't managed to do so.
I'll merge as it is right now, and keep that in a separated branch.
I'll give another try after I finish the xDai prices PR.

**Pull request merged**